code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -981,17 +922,17 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 		if(is_servant_of_ratvar(L))
 			servants++
 	if(servants * 0.2 < clockwork_daemons)
-		invoker << "<span class='nezbere'>\"Daemons are already disabled, making more of them would be a waste.\"</span>"
+		invoker << "<span class='nezbere'>\"Daemons aren't working. Can't give you another one.\"</span>"
 		return 0
-	if(servants * 0.2 < clockwork_daemons+1)
-		invoker << "<span class='nezbere'>\"This daemon would be useless, friend.\"</span>"
+	if(servants * 0.2 < clockwork_daemons + 1)
+		invoker << "<span class='nezbere'>\"Wouldn't do anything. They're useless right now.\"</span>"
```

Comment:
`It'd be useless.`, not `They're useless right now.`
This message only shows up if daemons ARE working, but making another would cause them to STOP.